tests/conftest.py

```diff
@@ -2,22 +2,109 @@
 import importlib
 import pytest
 import requests
+from sqlalchemy import create_engine
 import mock
 from unittest.mock import patch
 
 # indexd_server and indexd_client is needed as fixtures
-from cdisutilstest.code.indexd_fixture import clear_database
```

Comment:
i think that was the last use of it, so we should be able to remove the `cdisutilstest` dependency! 🎉